components/brave_referrals/browser/brave_referrals_service.cc

```diff
@@ -100,208 +58,30 @@ void BraveReferralsService::Start() {
                          base::Bind(&BraveReferralsService::GetFirstRunTime,
                                     base::Unretained(this)));
 
-  // Fetch the referral headers on startup.
-  FetchReferralHeaders();
-
-  // Also, periodically fetch the referral headers.
-  DCHECK(!fetch_referral_headers_timer_);
-  fetch_referral_headers_timer_ = std::make_unique<base::RepeatingTimer>();
-  fetch_referral_headers_timer_->Start(
-      FROM_HERE,
-      base::TimeDelta::FromSeconds(kFetchReferralHeadersFrequency +
-                                   base::RandInt(0, 60 * 10)),
-      this, &BraveReferralsService::OnFetchReferralHeadersTimerFired);
-  DCHECK(fetch_referral_headers_timer_->IsRunning());
-
   // On first run, read the promo code from user-data-dir and
   // initialize the referral.
   bool checked_for_promo_code_file =
       pref_service_->GetBoolean(kReferralCheckedForPromoCodeFile);
-  std::string download_id = pref_service_->GetString(kReferralDownloadID);
```

Comment:
Any references to download_id should stay, as that's how we track the referral in the browser.